xlml/utils/xpk.py

```diff
@@ -71,6 +69,7 @@ def run_workload(
             f" --command='{run_cmds}' --device-type={accelerator_type}"
             f" --num-slices={num_slices} --docker-image={docker_image}"
             f" --project={cluster_project} --zone={zone}"
+            f" --env {metric_config.SshEnvVars.GCS_OUTPUT.name}={gcs_path}"
```

Comment:
Does xpk has `--env` argument?

Reply:
I just added in https://github.com/google/xpk/pull/104, we'll have to wait for that one to land before this can merge.